rfc/networked-dot-storage.md

```diff
@@ -0,0 +1,174 @@
+
+It needs to exist for Filecoin AND it needs to exist for W3S.
+
+*** But isn't this a point of centralization if everyone uses it? ***
+
+Absolutely. It's important that key coordination functions in the network like a repair service eventually operate through distributed consensus and produce verifiable results.
+
+### Shared w3filecoin pipeline
+
+W3Filecoin is an excellent service for aggregating data on to Filecoin. At the same time, running it is NOT easy. While the basic aggregation functions are simple, making Filecoin deals is not. Multiple instances of W3UP should be able to share an instance of W3Filecoin. 
+
+*** But isn't this a point of centralization if everyone uses it? ***
+
+Absolutely. It's important that key coordination functions in the network like w3-filecoin eventually operate through distributed consensus and produce verifiable results.
+
+### Verification Mechanisms For Retrievals from W3UP Instances
+
+See [Proofs Overview](https://w3sat.notion.site/Filecoin-SPs-in-the-W3S-Network-16a9b66e7ef3425bbee390c6481bd752?pvs=4)
+
+It is important that the W3UP service level agreements are not compromised across the network by poorly performing or malicious W3Up instances.
+
+### Filecoin.Storage
+
+Produce an implementation of W3UP and W3Filecoin that can be run by a Filecoin L1 Storage Provider. In order words, a Filecoin storage provider should be able to be more than a backup on our network -- and eventually get seperate compensation -- if they can meet our service level requirements. See [Filecoin SPs in the W3S Network](https://w3sat.notion.site/Filecoin-SPs-in-the-W3S-Network-16a9b66e7ef3425bbee390c6481bd752?pvs=4)
+
+## FAQ
+
+### Is decentralization the wrong word to use?
+
+Decentralization is both a concept so overloaded it lacks any coherence, and a narrative that our product team has validated is indeed important to many market segments in web3. Actual customers have told us they won't use our product unless it's "decentralized". Likely, in some cases this is a novel kind of CYA (cover-your-ass) -- where enterprise customers want SOC-2 compliance in a data center so they can blame someone else if data gets lost, web3 customers want services they use to be "decentralized" so they can't get called out by the internet for not being "web3". Decentralization is both an annoying meme and an important narrative we have to contend with.
+
+I would say what we're really building is an incentivized permissionless cooperative distributed network for content addressed data. If anyone wants to try to market that, I encourage them to. But internally, maybe we should consider not saying decentralization as much. At minimum, let's align on what we're trying to achieve for our users.
```

Comment:
incentivized permissionless cooperative distributed network for content addressed data

IPCDNCAD.... it's got IP, CDN and CAD in it 🤣 

it is definitely wordy, but honestly I think this is a great starting point for thinking about how we do market this - different pieces will matter to different people but hey - more markets we can address!

Reply:
the idea of trying to market that phrase was intended as a joke :P

Reply:
but please, have a go.

Reply:
hahaha - it's probably a little too wordy to be a single marketing meme, but we should probably break it down and see what kinds of fun memetics we can get out of combinations of it - a cooperative network for content addressed data sounds dope as heck!